src/sdk/model/company/company.ts

```diff
@@ -530,7 +530,7 @@ export class Company extends Entity {
   async getLogo(): Promise<Uint8Array | null> {
     return Iland.getHttp().get(`/companies/${this.uuid}/logo`, {
       headers: {
-        'Accept': 'image/jpeg'
+        'Accept': 'image/vnd.ilandcloud.api.v1.0+jpeg'
```

Comment:
Lets go ahead and use the Http.versionMime method for this please

Reply:
oh i see - using now! looks like this is the first use of `Http.versionMime('image/jpeg')` so i added it to the `Http` unit tests as well

Reply:
Ah perfect thanks @iamtash !